src/main/java/com/aws/greengrass/device/ClientDevicesAuthServiceApi.java

```diff
@@ -45,7 +45,7 @@ public boolean verifyClientDeviceIdentity(String certificatePem) {
         if (deviceAuthClient.isGreengrassComponent(certificatePem)) {
             return true;
         } else {
-            Optional<String> certificateId = iotAuthClient.getActiveCertificateId(certificatePem);
+            Optional<String> certificateId = certificateRegistry.getIotCertificateIdForPem(certificatePem);
```

Comment:
We may still want to call the auth client directly here. What does `isThingAttachedToCertificate` return if a deactivated certificate is attached to the Thing?

Test:
* Connect client device to local broker
* Deactivate client certificate
* Ensure we can not re-connect again